src/api/bank-accounts/bank-account-approvals.md

```diff
@@ -35,6 +35,17 @@ An `approved` Bank Account Approval is required for the funds in a [Settlement W
 | modifiedAt    | {% dt Timestamp %} | When the Bank Account Approval was updated.                                                                            |
```

Comment:
@AndyClifford @MeganSteenkamp should there be an array of activities on this approval model?

Reply:
yep will add that now